sdk/dotnet/Pulumi.Automation/Workspace.cs

```diff
@@ -228,6 +228,22 @@ public Task SelectStackAsync(string stackName)
         /// </summary>
         public abstract Task<ImmutableList<StackSummary>> ListStacksAsync(CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Exports the deployment state of the stack.
+        /// <para/>
+        /// This can be combined with ImportStackAsync to edit a
+        /// stack's state (such as recovery from failed deployments).
+        /// </summary>
+        public abstract Task<StackDeployment> ExportStackAsync(string stackName, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Imports the specified deployment state into a pre-existing stack.
+        /// <para/>
+        /// This can be combined with ExportStackAsync to edit a
```

Comment:
You can `<see cref="ExportStackAsync(...)" />` here if you want to strongly reference the other method